tools/bundle-builder/index.js

```diff
@@ -112,8 +112,9 @@ function copyIcons() {
   // lerna bug: require.resolve() won't find spectrum-icons within spectrum-css-compat, so we have to assume it's at the top level
   // note: maybe it needs a package.main?
   return gulp.src([
-    `${dirs.topLevel}/node_modules/@spectrum/spectrum-icons/dist/svg/**`,
-    `${dirs.topLevel}/node_modules/@spectrum/spectrum-icons/dist/lib/**`
+    `${dirs.topLevel}/node_modules/@adobe/spectrum-css-workflow-icons/dist/**`,
+    `!${dirs.topLevel}/node_modules/@adobe/spectrum-css-workflow-icons/dist/18/**`,
```

Comment:
`dist/**` will cover all files and folders within `dist`, so the second two globs should not be necessary.

Reply:
But that two globs is to exclude the separated SVG icons to be copied to spectrum-css dist folder. Do we want to release them with spectrum-css?

Reply:
OH. No, we don't want to release them, thanks!